### PR TITLE
Use FQCN in examples

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: plugins/modules/container_remote.py [hunk 1]
```diff
@@ -75,11 +75,11 @@
 """
 
 RETURN = r"""
-  remotes:
+  pulp.squeezer.remotes:
```
Comment (evgeni) [inline plugins/modules/container_remote.py]: Return values have FQCNs now? ;)

Comment (mdellweg) [reply plugins/modules/container_remote.py]: :facepalm:

Comment (mdellweg) [reply plugins/modules/container_remote.py]: I would not risk a bet against it though.

**Comment by mdellweg** [comment]
What happened to the hash filter in ansible 2.8? I swear it worked before...

File: plugins/modules/ansible_distribution.py [hunk 2]
```diff
@@ -81,11 +81,11 @@
 """
 
 RETURN = r"""
-  distributions:
+  pulp.squeezer.distributions:
     description: List of ansible distributions
     type: list
     returned: when no name is given
-  distribution:
+  pulp.squeezer.distribution:
```
Comment (evgeni) [inline plugins/modules/ansible_distribution.py]: no

**Review by evgeni — CHANGES_REQUESTED** [review]
ansible_distribution.py is still fucked up ;)

**Review by evgeni — DISMISSED** [review]

File: plugins/modules/status.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
 
 EXAMPLES = r"""
 - name: Read status from pulp api server
-  status:
+  pulp.squeezer.status:
     pulp_url: localhost:24817
```
Comment (evgeni) [inline plugins/modules/status.py:28]: unrelated, but I think you could unify the `pulp_url`s ;)

**Review by evgeni — APPROVED** [review]

File: plugins/modules/status.py (hunk 3 above)
Comment (mdellweg) [reply plugins/modules/status.py:28]: Yes, i could.
https://github.com/pulp/squeezer/pull/98

**Pull request merged**